velox/core/QueryConfig.h

```diff
@@ -102,6 +111,16 @@ class QueryConfig {
     return get<uint64_t>(kMaxPartialAggregationMemory, kDefault);
   }
 
+  uint64_t maxExtendedPartialAggregationMemoryUsage() const {
+    static constexpr uint64_t kDefault = 1L << 24;
```

Comment:
So we are not planning to expand it more than max partial?

Reply:
Let's keep the default behavior the same. And bump up it later once tuning in staging env for a while?

Reply:
We don't have prod now. Actually you can treat our current Deltoid as staging.

Reply:
Yeah, we can try on staging per our offline discussion. Thanks!